src/dotnet/commands/dotnet-list/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Tools.List.ProjectToProjectReferences;
+
+namespace Microsoft.DotNet.Tools.List
+{
+    public class ListCommand : DispatchCommand
+    {
+        protected override string HelpText => $@"{LocalizableStrings.ListCommandDescription}
+
+{LocalizableStrings.Usage}: dotnet list [options] <object> <command> [[--] <arg>...]]
+
+Options:
+  -h|--help  {LocalizableStrings.HelpDefinition}
+
+{LocalizableStrings.Arguments}:
+  <object>   {LocalizableStrings.ObjectDefinition}
+  <command>  {LocalizableStrings.CommandDefinition}
```

Comment:
we do - this is inside `dotnet list` which is a metacommand and in fact does execute other commands (probably in the future will be extensible) - I believe you got confused with dotnet-list-p2ps command which in fact does not do that